hs_core/hydroshare/utils.py

```diff
@@ -238,6 +186,7 @@ def get_federated_zone_home_path(filepath):
     Returns:
         the zone name extracted from filepath
     """
+    # TODO: perhaps add HS_IRODS_PROXY_USER_IN_USER_ZONE back in
```

Comment:
I think this comment isn't needed anymore?

Reply:
yes, thanks removed it